### ISSUE TITLE
Missing keyboard accessibility for Select

### ISSUE THREAD

**Issue by gw2princeps** [N00nDay/stwui#115]
According to the WebAIM org it should be possible to navigate options of the select component with the key up and key down buttons. 

https://webaim.org/techniques/keyboard/ 

Potentially, there are more violations, I'll report back in case I notice more issues.

**Comment by N00nDay** [comment]
I am expanding this to the following components:

- [X] DatePicker
- [X] Select
- [ ] Autocomplete
- [ ] Dropdown
- [ ] Radio
- [ ] Checkbox
